### ISSUE TITLE
Todo

### ISSUE THREAD

**Issue by PKuznetsov-vl** [PKuznetsov-vl/logreg#1]
Ced graph
Loss graph - for all loss 
functions to watch their efficiency
Classes plot utils 
Как правильно 
https://keras.io/guides/sequential_model/

https://www.tensorflow.org/guide/keras/custom_layers_and_models

Связать с этим 
https://disk.yandex.ru/i/85iVdhaPDKufOA
